pkg/volume/csi/csi_plugin.go

```diff
@@ -216,12 +218,74 @@ func (p *csiPlugin) Init(host volume.VolumeHost) error {
 		go factory.Start(wait.NeverStop)
 	}
 
+	var migratedPlugins = map[string](func() bool){
+		csitranslationplugins.GCEPDInTreePluginName: func() bool {
+			return utilfeature.DefaultFeatureGate.Enabled(features.CSIMigration) && utilfeature.DefaultFeatureGate.Enabled(features.CSIMigrationGCE)
+		},
+		// TODO(leakingtpan): Add AWS migration feature gates and place them here
```

Comment:
do AWS specifics have to exist under `pkg/volume/csi/csi_plugin.go`?

Reply:
yes, there will need to be specifics for each in-tree plugin. Unfortunately with migration we need a 1-1 mapping from in-tree to CSI and so there will need to be some specific logic baked into CSI